makefiles/scan-build.inc.mk

```diff
@@ -70,13 +91,9 @@ scan-build-analyze: clean
 	$(Q)mkdir -p '$(SCANBUILD_OUTPUTDIR)'
 	$(Q)env -i $(ENVVARS) \
 	    scan-build -o '$(SCANBUILD_OUTPUTDIR)' $(SCANBUILD_ARGS) \
-	      make -C $(CURDIR) all $(strip $(CMDVARS)) FORCE_ASSERTS=1;
-endif # BUILD_IN_DOCKER
+	      make -C $(CURDIR) all $(strip $(CMDVARS)) FORCE_ASSERTS=1
 
-ifeq (1,$(INSIDE_DOCKER))
-scan-build-view:
```

Comment:
It was not working before, you could not do `scan-build-view` without rebuilding everything anyway. So was basically the same as `scan-build`.
Making it work is a new feature.

I could re-add a separate `scan-build-view` that does not rebuild by duplicating the view command (== putting as a function)
It is more a bash script implementation here as there is no file targets that can be up to date or not, so I am not sure there is a nice way around duplicating it.

I could try something so that the `scan-build-view` only builds if there is no report.


Do you want me to try something for this ?

Reply:
Ok, you are free to try, but let's do it then in a follow-up PR.